sdk/tables/azure-tables/review/tables.api.md

```diff
@@ -106,7 +106,7 @@ export interface GeoReplication {
 }
 
 // @public
-export type GeoReplicationStatusType = "live" | "bootstrap" | "unavailable";
+export type GeoReplicationStatusType = "live" | "bootstrap" | "unavailable" | string;
```

Comment:
FYI - The `GeoReplication` interface type is undocumented (hidden slightly above this type in the diff).

Reply:
Added a transform to add docs to these